packages/widget/src/providers/WidgetProvider/WidgetProvider.tsx

```diff
@@ -29,45 +27,9 @@ export const WidgetProvider: React.FC<
 
   const value = useMemo((): WidgetContextProps => {
     try {
-      // Get search params from URL
-      const searchParams = Object.fromEntries(
-        new URLSearchParams(window?.location.search),
-      );
-      // Prevent using fromToken/toToken params if chain is not selected
-      ['from', 'to'].forEach((key) => {
-        if (searchParams[`${key}Token`] && !searchParams[`${key}Chain`]) {
-          delete searchParams[`${key}Token`];
-        }
-      });
```

Comment:
The logic here has been moved to the URLSearchParamsBuilder and now rather than overridding the config values is used to set form values in the form store

https://github.com/lifinance/widget/blob/fbbd18c4a1e846e0570417b67b12ac47157418bb/packages/widget/src/stores/form/URLSearchParamsBuilder.tsx#L59-L68

and 

https://github.com/lifinance/widget/blob/fbbd18c4a1e846e0570417b67b12ac47157418bb/packages/widget/src/stores/form/URLSearchParamsBuilder.tsx#L19-L45